rotkehlchen/detail_mappings.py

```diff
@@ -0,0 +1,54 @@
+from rotkehlchen.types import Location, LocationDetails
```

Comment:
This should not be in the root of the project. Put it under `/constants`. Also name can be more specific. location_details.py